client/components/Medical-Contact/ContactPage.vue

```diff
@@ -13,9 +13,17 @@
     <section>
       <header>
         <h2>Current Contacts</h2>
+        <div class="search-wrapper">
+          <label>Search by contact last name:</label>
+          <input
```

Comment:
I wonder if it'd be worth the effort to make contacts searchable by any part of the name, not just last name?